easybuild/easyconfigs/p/Perl/Perl-5.28.1-GCCcore-8.2.0.eb

```diff
@@ -1495,6 +1495,251 @@ exts_list = [
         'source_urls': ['https://cpan.metacpan.org/authors/id/J/JD/JDHEDDEN/'],
         'checksums': ['9dfd6ca2822724347e0eb6759d00709425814703ad5c66bdb6214579868bcac4'],
     }),
+    ('ExtUtils::MakeMaker', '7.36', {
```

Comment:
@lexming Do you mind fleshing this out in a separate PR, and adding the same extensions to `Perl-5.30.0-GCCcore-8.3.0.eb` as well?

Reply:
Sure, I've moved those changes to #10049